test/unit-tests/MultiBridgeMessageSender.t.sol

```diff
@@ -217,24 +249,42 @@ contract MultiBridgeMessageSenderTest is Setup {
         vm.startPrank(caller);
         vm.expectRevert(Error.INVALID_EXPIRATION_DURATION.selector);
         sender.remoteCall(
-            DST_CHAIN_ID, address(42), bytes("42"), 0, invalidExpMin, refundAddress, fees, excludedAdapters
+            DST_CHAIN_ID, address(42), bytes("42"), 0, invalidExpMin, refundAddress, fees, DEFAULT_SUCCESS_THRESHOLD,
+            new address[](0)
         );
 
         vm.expectRevert(Error.INVALID_EXPIRATION_DURATION.selector);
         sender.remoteCall(
-            DST_CHAIN_ID, address(42), bytes("42"), 0, invalidExpMax, refundAddress, fees, excludedAdapters
+            DST_CHAIN_ID, address(42), bytes("42"), 0, invalidExpMax, refundAddress, fees, DEFAULT_SUCCESS_THRESHOLD,
+            new address[](0)
```

Comment:
There are several instances like this where we recreate an empty array to pass to `remoteCall()` for the exclusion list. These functions already have a declared `excludedAdapters` variable that is set to an empty array for this purpose. Suggest reusing that consistently.